src/NUnitToXUnit/Visitor/CompilationUnitVisitor.cs

```diff
@@ -22,6 +22,8 @@ public NUnitToXUnitVisitor(IConverterOptions options)
         public override SyntaxNode VisitCompilationUnit(CompilationUnitSyntax node)
         {
             var xunitTree = (CompilationUnitSyntax)base.VisitCompilationUnit(node);
+            var leadingTrivia = xunitTree.GetLeadingTrivia();
+            xunitTree = xunitTree.WithoutLeadingTrivia();
```

Comment:
what is this doing? could you add some comments?

Reply:
Resolved in 4eccbb8